addons/sale/views/sale_portal_templates.xml

```diff
@@ -442,6 +442,9 @@
                                     <span t-if="i.payment_state in ('paid', 'in_payment')" class="small badge text-bg-success orders_label_text_align">
                                         <i class="fa fa-fw fa-check"/> <b>Paid</b>
                                     </span>
+                                    <span t-elif="i.payment_state == 'reversed'" class="small badge badge-success orders_label_text_align">
+                                        <i class="fa fa-fw fa-check"/> <b>Reversed</b>
+                                    </span>
```

Comment:
Hello @Demesmaeker and @Feyensv :wave: 

Should `Reversed` badge be below `Authorized` badge or above?
I think it should be above but asking to be sure :smiley:

Thank you :slightly_smiling_face: